package.json

```diff
@@ -68,7 +68,7 @@
   "scripts": {
     "check": "node -r dotenv/config bin/check.js",
     "run": "node -r dotenv/config index.js",
-    "test": "mocha tests/**_test.js -R ./node_modules/@testomatio/reporter/lib/adapter/mocha.js",
```

Comment:
are you sure this will work?

Reply:
Yep, I have tested it before push. And If you try to use `-R ./node_modules/@testomatio/reporter/lib/adapter/mocha.js` command => no test results.